### ISSUE TITLE
JTextPane always auto scrolls to end of file when making a change.

### ISSUE THREAD

**Issue by raquel-ucl** [oracc/nammu#83]

**Comment by raquel-ucl** [comment]
This happens because when the modification of the ATF edit area adds/removes lines, the line numbers panel is repainted, and leaves the caret at the end of that panel, which causes the JScrollPane container to scroll down.
Could be fixed by trying to prevent the caret in the line numbers panel from updating like in this JTextArea example:

```
JTextArea textArea = new JTextArea();
DefaultCaret caret = (DefaultCaret)textArea.getCaret();
caret.setUpdatePolicy(DefaultCaret.NEVER_UPDATE);
```

**Comment by raquel-ucl** [comment]
Fixed at #73 (d90e78d)
